packages/browser-pool/test/tsconfig.json

```diff
@@ -4,6 +4,5 @@
 		"noEmit": true,
 		"incremental": false,
 		"types": ["vitest/globals"]
-	},
-	"include": ["./"]
```

Comment:
this seems unrelated :eyes: 

But I suppose the `tsconfig.json` default for this option is `**/*` anyway (relative to the `tsconfig.json` parent dir), so :shrug: